
 make.py started: 2016-08-01 19:13:27 /Users/Frank/Desktop/gslab_python/gslab_fill 


Path: gslab_make
URL: https://econ-gentzkow-svn.stanford.edu/repos/main/trunk/lib/python/gslab_make
Repository Root: https://econ-gentzkow-svn.stanford.edu/repos/main
Repository UUID: 89b4331b-4301-411a-8e06-9c78d9cdb83d
Revision: 34789
Node Kind: directory
Last Changed Author: jwchoi2
Last Changed Rev: 33502
Last Changed Date: 2016-02-07 17:40:17 -0800 (Sun, 07 Feb 2016)



Execute:  statamp -e do "logs_for_textfill.do"
/bin/sh: statamp: command not found
ERROR! Command statamp -e do "logs_for_textfill.do" executed with errors 
Command 'statamp -e do "logs_for_textfill.do"' returned non-zero exit status 127


Traceback (most recent call last):
  File "/Library/Python/2.7/site-packages/gslab_make/py/run_program.py", line 52, in run_stata
    run.move_log(default_log)
  File "/Library/Python/2.7/site-packages/gslab_make/py/private/runprogramdirective.py", line 238, in move_log
    raise CritError(messages.crit_error_no_file % default_log)
CritError: 'ERROR! File ./test/input/logs_for_textfill.log not found'


Execute:  python  "run_all_tests.py" 
testArgumentOrder (test_tablefill.testTablefill) ... ok
testBreaksRoundingString (test_tablefill.testTablefill) ... ok
testIllegalSyntax (test_tablefill.testTablefill) ... ok
testInput (test_tablefill.testTablefill) ... ok
testAlternativePrefix (test_textfill.testTextfill) ... ok
testArgumentOrder (test_textfill.testTextfill) ... ok
testIllegalSyntax (test_textfill.testTextfill) ... ok
testInput (test_textfill.testTextfill) ... ok
testRemoveEchoes (test_textfill.testTextfill) ... ok
testTagsDontMatch (test_textfill.testTextfill) ... ok
testTagsIncorrectlySpecified (test_textfill.testTextfill) ... ok
testTagsNotClosed (test_textfill.testTextfill) ... ok

----------------------------------------------------------------------
Ran 12 tests in 0.074s

OK

 make.py ended: 2016-08-01 19:13:29
